v1/src/main/java/com/google/cloud/teleport/templates/DatastoreToBigQuery.java

```diff
@@ -79,6 +79,8 @@ public static void main(String[] args) {
             TransformTextViaJavascript.newBuilder()
                 .setFileSystemPath(options.getJavascriptTextTransformGcsPath())
                 .setFunctionName(options.getJavascriptTextTransformFunctionName())
+                .setReloadFunction(options.getJavascriptFunctionReload())
+                .setReloadIntervalMinutes(options.getJavascriptReloadIntervalMinutes())
```

Comment:
I don't think it makes a lot of sense for batch pipelines

Reply:
Explaining: batch jobs are short-lived by definition, I think having a single runtime for every run makes things simpler / predictable

Reply:
Removed